app/templates/components/post-header.hbs

```diff
@@ -1,2 +1,2 @@
 <div class="post-icon"></div>
-{{post-title post=post}}
+{{post-title post=post saveTitle=saveTitle}}
```

Comment:
Maybe not for comments (that's a different beast), but for anything that needs saved on the post, yes.